File: Algorithms/Searching/python/linerSearch.py
def search(arr, t):
    for i in range(0, len(arr)):
        if arr[i] == t:
            print("FOUND")
            return
    print("NOT FOUND")
    return


if __name__ == "__main__":
    arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
    search(arr, 4)

File: Algorithms/Searching/cpp/linerSearch.cpp
#include <iostream>

using namespace std;

int search(int arr[], int target) {

  for (int i = 0; i < 10; i++) {
    if (arr[i] == target) {
      std::cout << "Found";
      return 1;
    }
  }
  std::cout << "Not Found";
  return 0;
}

int main() {

  int a[10] = {1, 2, 14, 5, 6, 2, 4, 2, 10, 99};

  search(a, 99);

  return 1;
}
